prow/plugins/plugins.go

```diff
@@ -431,8 +431,8 @@ func (pa *ConfigAgent) getPlugins(owner, repo string) []string {
 	var plugins []string
 
 	fullName := fmt.Sprintf("%s/%s", owner, repo)
-	plugins = append(plugins, pa.configuration.Plugins[owner]...)
-	plugins = append(plugins, pa.configuration.Plugins[fullName]...)
+	plugins = append(plugins, pa.configuration.Plugins()[owner].Plugins...)
+	plugins = append(plugins, pa.configuration.Plugins()[fullName].Plugins...)
```

Comment:
Shouldn't we process excluded repos here?

Reply:
yes